lib/gc/ExecutionEngine/CPURuntime/Microkernel/BrgemmOnednn.cpp

```diff
@@ -57,6 +57,53 @@ static std::vector<brgemm_desc_t> g_brgemm_desc_list;
 static std::vector<brgemm_kernel_t *> g_brgemm_kernel_list;
 static std::vector<std::unique_ptr<char[]>> g_brgemm_palette;
 
+struct brgemm_cache_info_t {
+  std::shared_ptr<brgemm_desc_t> desc;
+  std::shared_ptr<brgemm_kernel_t> kernel;
+  std::shared_ptr<char> palette;
```

Comment:
As we created `brgemm_cache_info_t` to store desc/kernel/palette together for thread locally, would it be a better manner to also use `brgemm_cache_info_t` for global management?

Reply:
I think it's a good idea, we can unify the struct used in both thead-local and global